Sources/TokamakDOM/Resources/TokamakStyles.swift

```diff
@@ -24,4 +24,11 @@ let tokamakStyles = """
   width: 0;
   height: 0;
 }
+._tokamak-list {
+  list-style: none;
+  overflow-y: scroll;
```

Comment:
```suggestion
  overflow-y: auto;
```

on systems where scrollbars are always visible, `scroll` shows empty scroll tracks even if the list doesn’t need to scroll.